ghost/i18n/locales/tr/portal.json

```diff
@@ -25,12 +25,12 @@
     "Cancel subscription": "Aboneliği iptal et",
     "Cancellation reason": "İptal sebebi",
     "Change": "Değiştir",
-    "Change plan": "",
+    "Change plan": "Plan Değiştir",
```

Comment:
Sorry, one more with the same capitalization quirk.

Reply:
fixed